map2loop/sampler.py

```diff
@@ -84,6 +84,7 @@ def sample(
         data = spatial_data.copy()
         data["X"] = data.geometry.x
         data["Y"] = data.geometry.y
+        map_data.get_value_from_raster_df(Datatype.DTM, data)
```

Comment:
Inplace modification of arrays and dataframes should be avoided. Better to return a new dataframe with the updated values

Reply:
in https://github.com/Loop3D/map2loop/pull/91/commits/722b98cdbcf1f820b9d1a480f09ac59d287f1a04